src/components/binding/binding-loader.js

```diff
@@ -113,6 +118,29 @@ genotet.BindingLoader.prototype.loadFullTrack = function(trackIndex, fileName,
 };
 
 /**
+ * Loads the bed data of a single binding track.
+ * @param {string} bedName Bed name of the bed binding track.
```

Comment:
Use fileName so that it matches with server.
https://github.com/ViDA-NYU/genotet/blob/refactor/server/bed.js